frontend/src/__tests__/cypress/cypress/utils/oc_commands/project.ts

```diff
@@ -71,3 +71,43 @@ export const addUserToProject = (
     return result;
   });
 };
+
+/**
+ * Verify if an OpenShift Project exists
+ *
+ * @param projectName OpenShift Project name to verify (in the format like 'cypress-test-project')
+ * @returns A Cypress chainable boolean indicating whether the project exists (true) or not (false)
+ */
+export const verifyOpenShiftProjectExists = (projectName: string): Cypress.Chainable<boolean> => {
+  if (!projectName || typeof projectName !== 'string') {
+    cy.log(`ERROR: Invalid project name provided: ${projectName}`);
+    throw new Error(`Invalid project name: ${projectName}`);
+  }
+
+  const checkCommand = `oc get project "${projectName}" -o name`;
+
+  cy.log(`Executing command: ${checkCommand}`);
+
+  return cy.exec(checkCommand, { failOnNonZeroExit: false }).then((result: Cypress.Exec) => {
+    cy.log(`Command result: ${JSON.stringify(result)}`);
+
+    if (result.code !== 0 && result.code !== 1) {
+      // Code 1 is expected when the project doesn't exist, so we only throw for other non-zero codes
+      cy.log(`ERROR: Command execution failed
+              stdout: ${result.stdout}
+              stderr: ${result.stderr}`);
+      throw new Error(`Command failed with code ${result.code}`);
+    }
+
+    const projectExists =
+      result.code === 0 && result.stdout.trim() === `project.project.openshift.io/${projectName}`;
+
+    if (projectExists) {
+      cy.log(`Project '${projectName}' exists.`);
+    } else {
+      cy.log(`Project '${projectName}' does not exist.`);
+    }
+
+    return cy.wrap(projectExists);
+  });
+};
```

Comment:
```suggestion
/**
 * Get OpenShift Project 
 *
 * @param projectName OpenShift Project name to get (in the format like 'cypress-test-project')
 * @returns A Cypress chainable with the project information or null if not found
 */
export const getOpenShiftProject = (projectName: string): Cypress.Chainable<string | null> => {
  if (!projectName || typeof projectName !== 'string') {
    cy.log(`ERROR: Invalid project name provided: ${projectName}`);
    throw new Error(`Invalid project name: ${projectName}`);
  }

  const checkCommand = `oc get project "${projectName}" -o name`;

  cy.log(`Executing command: ${checkCommand}`);

  return cy.exec(checkCommand, { failOnNonZeroExit: false }).then((result: Cypress.Exec) => {
    cy.log(`Command result: ${JSON.stringify(result)}`);

    if (result.code !== 0 && result.code !== 1) {
      // Code 1 is expected when the project doesn't exist, so we only throw for other non-zero codes
      cy.log(`ERROR: Command execution failed
              stdout: ${result.stdout}
              stderr: ${result.stderr}`);
      throw new Error(`Command failed with code ${result.code}`);
    }

    return result.code === 0 ? result.stdout.trim() : null;
  });
};

/**
 * Verify if an OpenShift Project exists
 *
 * @param projectName OpenShift Project name to verify (in the format like 'cypress-test-project')
 * @returns A Cypress chainable boolean indicating whether the project exists (true) or not (false)
 */
export const verifyOpenShiftProjectExists = (projectName: string): Cypress.Chainable<boolean> => {
  return getOpenShiftProject(projectName).then((projectInfo) => {
    const projectExists = projectInfo === `project.project.openshift.io/${projectName}`;

    if (projectExists) {
      cy.log(`Project '${projectName}' exists.`);
    } else {
      cy.log(`Project '${projectName}' does not exist.`);
    }

    return cy.wrap(projectExists);
  });
};
```

Reply:
Maybe it's better approach split this method in two:
1. Getting the DSP (which is going to be reusable)
2. Do the verification (which maybe should not go here as not contains anything oc related)

Reply:
@antowaddle before I split this up, would there be any scenario where we would only want to do a Verification that wouldn't require getting the project first?

Reply:
@antowaddle I'm not sure we want to make a  verifyOpenShiftProjectExists that doesn't contain the get. In fact in the example I have proposed, in line 113 the get is called. 
What I do see probable is that, at some point, we want to make use of the get, without it being inside the verify. Hence my proposal to divide them in two parts:
1. Get
2. Verify (which uses the Get)